python/lsst/ts/observatory/control/data/atcalsys_schema.yaml

```diff
@@ -4,48 +4,105 @@ title: Configuration file validation schema of Auxiliary Telescope Calibration S
     type: string
   wavelength:
+    description: >-
+      Center wavelength value in nm, used for configuring the monochromator
+      output.
     type: number
   wavelength_width:
+    description: >-
+      Optional. Defines the width of the wavelength scan range to configure the
+      monochromator for the flat-field calibration sequence when using
+      monochromatic light.
     type: number
   wavelength_resolution:
+    description: >-
+      Optional. When using a monochromatic light source, it defines the
+      resolution used to sample values within the scan range, which has a width
+      defined by `wavelength_width` and is centered around the `wavelength`
+      attribute.
     type: number
   monochromator_grating:
+    description: >-
+      Select the grating of the monochromator that will be in the path of
+      light.
     type: string
     enum:
       - MIRROR
       - RED
       - BLUE
   exit_slit:
+    description: >-
+      Exit slit width for the monochromator, from 0 to 7 mm (inclusive).
     type: number
   entrance_slit:
+    description: >-
+      Entrance slit width for the monochromator, from 0 to 7mm (inclusive).
     type: number
   electrometer_integration_time:
+    description: >-
+      Integration time in seconds (166.67e-6 to 200e-3) for each sample.
+
+      The integration time (measurement speed) of the analog to digital (A/D)
+      converter, the period of time the input signal is measured (also known as
+      aperture). Due to the time it takes to read the buffer and process the
+      data, this is not the rate at which samples are taken. This is generally
+      specified by the Power Line Cycle (PLC), where 1 PLC for 60Hz is
+      16.67msec. Fast integration=0.01PLC; Medium integration=0.1PLC; Normal
+      (default) integration = 1PLC; High Accuracy integration=10PLC. Here the
+      integration is set in seconds.
     type: number
   electrometer_mode:
+    description: >-
+      Set electrometer to use different modes. The units recorded will be Amps
+      for `CURRENT`, Volts for `VOLTAGE`, Coulombs for `CHARGE`, and Ohms for
+      `RESISTANCE`.
     type: string
     enum:
       - CURRENT
       - CHARGE
       - VOLTAGE
       - RESISTANCE
   electrometer_range:
+    description: >-
+      Set measurement range, which effects the accuracy of measurements and the
+      max signal that can be measured. The maximum input signal is 105% of the
+      measurement range. It will set the value for the current mode selected.
+      Auto-range will automatically go to the most sensitive (optimized) range
+      to make measurements. It is recommended to use autorange. When the
+      measurement range is changed, a zero correction will be performed.
+
+      -1 for automatic range. Volts range from 0 to 210 Volts, Current range
+      from 0 to 21e-3 Amps, Resistance from 0 to 100e18 Ohms, Charge from 0 to
+      +2.1e-6 Coulombs.
     type: number
```

Comment:
Again, it may have too much information. Let me know if it should be summarized.